lib/netconf/static.go

```diff
@@ -135,3 +135,8 @@ func ConsensusChainIDStr2Uint64(id string) (uint64, error) {
 
 	return resp, nil
 }
+
+// IsOmniConsensus returns true if provided chainID is the omni consensus chain for the network.
+func IsOmniConsensus(network ID, chainID uint64) bool {
```

Comment:
Matter of preference but I would prefer to define this as:

`func IsOmniConsensus(network network, chainID chain) bool { ... }`

That way it is always called the same way throughout the codebase

Reply:
one doesn't always have a network, sometimes you only have ID